src/Sodoku.java

```diff
@@ -1,126 +1,38 @@
-import java.awt.PageAttributes.PrintQualityType;
-import java.util.ArrayList;
+import java.util.*;
 
 public class Sodoku {
 
 	public static void main(String[] args) {
-		Puzzle puzzle = new Puzzle();
-		// puzzleOne(puzzle);
-		puzzleTwo(puzzle);
+		// Puzzle puzzle = new Puzzle(new String[]{
+		// 	" 1    4  ",
+		// 	"3421 58 9",
+		// 	"697      ",
+		// 	"7   635 4",
+		// 	" 36      ",
+		// 	"8   216  ",
+		// 	" 8  172  ",
+		// 	"4 158293 ",
+		// 	" 7  94  5"
+		// });
+		Puzzle puzzle = new Puzzle(new String[]{
+			"73   5   ",
+			" 4  6    ",
```

Comment:
Thsi new constructor just seems so much easier to work with than having to call setValue dozens of times.